app/controllers/support/providers/accredited_provider_search_controller.rb

```diff
@@ -17,6 +17,13 @@ def create
           @accredited_provider_select_form = AccreditedProviderSelectForm.new
           @accredited_provider_search = ::AccreditedProviders::SearchService.call(query:)
 
+          if @accredited_provider_search.providers.size == 1
```

Comment:
So without js this journey will be skipped?

![image](https://github.com/DFE-Digital/publish-teacher-training/assets/470137/ea07918e-9a17-4bb1-8cd8-04ceff9bc2bf)

Reply:
That was the info I had from @darokel but I will check.

Reply:
The radio list is to be skipped